client/src/commands/addWalletIdentityCommand.ts

```diff
@@ -197,7 +193,33 @@ export async function addWalletIdentity(walletItem: WalletTreeItem | IFabricWall
 
             const certificateAuthority: IFabricCertificateAuthority = FabricCertificateAuthorityFactory.createCertificateAuthority();
 
-            const enrollment: {certificate: string, privateKey: string} = await certificateAuthority.enroll(gatewayRegistryEntry.connectionProfilePath, enrollmentID, enrollmentSecret);
+            // Read connection profile
+            const connectionProfileFile: string = await fs.readFile(gatewayRegistryEntry.connectionProfilePath, 'utf8');
+            let connectionProfile: any;
+
+            if (gatewayRegistryEntry.connectionProfilePath.endsWith('.json')) {
+                connectionProfile = JSON.parse(connectionProfileFile);
+            } else {
+                // Assume its a yml/yaml file type
+                connectionProfile = yaml.safeLoad(connectionProfileFile);
+            }
+
+            // Get a list of CAs
+            const caKeys: any[] = Object.keys(connectionProfile.certificateAuthorities);
```

Comment:
Could you make the type `string[]` please

Reply:
Done